recipe/meta.yaml

```diff
@@ -109,7 +108,7 @@ outputs:
         - PIP_CACHE_DIR=pip_cache
         - PIP_NO_INDEX=True
     requirements:
-      build:
+      build:                                  # [build_platform != target_platform]
         - python                              # [build_platform != target_platform]
         - cross-python_{{ target_platform }}  # [build_platform != target_platform]
```

Comment:
Hiding the `requirements/build` section completely when unused. Otherwise we see this error (though the error doesn't appear to stop the build)

```
EnvironmentLocationNotFound: Not a conda environment: D:\bld\xgboost-split_1727635034975\_build_env
```